docs/openapi.yaml

```diff
@@ -213,6 +213,56 @@ paths:
               example:
                 $ref: ./api/transaction/get-mempool-transactions.example.json
 
+  /extended/v1/tx/multiple:
+    parameters:
+      - name: tx_id
+        in: query
+        description: Array of transaction ids
+        required: true
+        schema:
+          type: array
+      - name: event_offset
+        in: query
+        schema:
+          type: integer
+          default: 0
+        description: The number of events to skip
+      - name: event_limit
+        in: query
+        schema:
+          type: integer
+          default: 96
+        description: The numbers of events to return
+      - name: unanchored
+        in: query
+        description: Include transaction data from unanchored (i.e. unconfirmed) microblocks
+        required: false
+        schema:
+          type: boolean
+          default: false
+    get:
+      summary: Get list of details for transactions
+      tags:
+        - Transactions
+      operationId: get_tx_list_details
+      description: |
+        Get an array of transactions by IDs
+
+        If using TypeScript, import typings for this response from our types package:
+
+        `import type { Transaction } from '@stacks/stacks-blockchain-api-types';`
+      responses:
+        200:
+          description: Returns list of transactions with their details for corresponding requested tx_ids.
+          content:
+            application/json:
+              schema:
+                $ref: ./entities/transactions/transaction.schema.json
```

Comment:
The response schema should be an array of tx results right?

Reply:
You're right. My bad. Created return type for this response and updated open.yaml.